examples/sandbox/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
     "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
This makes TS happy when you do `import $ from "jquery"`

Reply:
yeah this option makes sense :+1: